homework_8/cat.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python3
```

Comment:
Верно, только без пробела
```suggestion
#!/usr/bin/env python3
```